src/main/java/ee/tuleva/onboarding/payment/email/PaymentEmailSender.java

```diff
@@ -25,6 +25,8 @@
 
 @Component
 @RequiredArgsConstructor
+// TODO: Rename to third pillar payment email sender
+// TODO: or is this event based implementation even needed?
```

Comment:
Todo comments. What's the question here exactly?

Reply:
Discussed during call, removing these comments